src/Tasks/ManifestUtil/ManifestReader.cs

```diff
@@ -224,7 +224,9 @@ public static Manifest ReadManifest(string manifestType, Stream input, bool pres
         private static Manifest Deserialize(Stream s)
         {
             s.Position = 0;
-            using var r = new XmlTextReader(s) { DtdProcessing = DtdProcessing.Ignore };
+#pragma warning disable CA2000 // Dispose objects before losing scope is suppressed because the stream will be handled in the caller
+            var r = new XmlTextReader(s) { DtdProcessing = DtdProcessing.Ignore };
```

Comment:
I think this can be fixed by using `XmlReader.Create` with `XmlReaderSettings.CloseInput` set to false. But if you're in a hurry, this is fine too.